src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -224,20 +225,19 @@ public String exportSnapshot(
 
   /**
    * @param userReq authenticated user
+   * @param errors list to store any exceptions encountered
    * @return accessible snapshot UUIDs and the IamRoles dictating their accessibility, established
    *     directly by SAM and indirectly by any linked RAS passport
    */
-  public Map<UUID, Set<IamRole>> listAuthorizedSnapshots(AuthenticatedUserRequest userReq) {
+  public Map<UUID, Set<IamRole>> listAuthorizedSnapshots(
```

Comment:
If this is no longer called from `SnapshotsApiController` then it no longer needs to be public. Using default access and marking it `@VisibleForTesting` may be better.

Reply:
Good point, thanks!